eg/Utils.py

```diff
@@ -394,23 +394,23 @@ def IsVista():
     """
     warnings.warn(
         "eg.Utils.IsVista() is deprecated. "
-        "Use eg.WindowsVersion.IsVista() instead",
+        "Use eg.WindowsVersion >= 'Vista' instead",
         DeprecationWarning,
         stacklevel=2
     )
-    return eg.WindowsVersion.IsVista()
+    return eg.WindowsVersion >= 'Vista'
 
 def IsXP():
     """
     Determine if we're running XP or higher.
     """
     warnings.warn(
         "eg.Utils.IsXP() is deprecated. "
-        "Use eg.WindowsVersion.IsXP() instead",
+        "Use eg.WindowsVersion >= 'XP' instead",
```

Comment:
The return value of `eg.Utils.IsVista()` is correct. I only meant the deprecation message. Just to make sure that the user/developer notice that there is a difference behaviour of `eg.Utils.IsVista()` and `eg.WindowsVersion.IsVista()`.
But now while i write this, i think you are right and only display `>= 'Vista'` because that's how Utils.IsVista() worked.
 I will merge it now and make a new release.